deployment/compose.yml

```diff
@@ -10,24 +11,28 @@ services:
       - 8222:8222
 
   zambeze_agent1:
+    environment:
+      NATS_HOST: nats
+      NATS_PORT: 4222
+      PYTHONWARNINGS: "ignore::DeprecationWarning"
     container_name: zambeze1
     image: zambeze/zambeze:latest
     command: [/bin/sh, -c, 'zambeze agent start & sleep 2 && tail -f /root/.zambeze/logs/`ls -t /root/.zambeze/logs/ | head -n 1`']
     depends_on:
       -  nats
+
+  zambeze_agent2:
     environment:
       NATS_HOST: nats
       NATS_PORT: 4222
-
-  zambeze_agent2:
+      PYTHONWARNINGS: "ignore::DeprecationWarning"
     container_name: zambeze2
     image: zambeze/zambeze:latest
-    tty: true
-    stdin_open: true
-    volumes: 
+    volumes:
       - ../db:/zambeze/db
-      - ../:/zambeze  # This line is needed only for development.
+      - ../:${CONTAINER_VOL_MNT_POINT:-/zambeze}  # This line is needed only to facilitate development.
```

Comment:
Where in the CI is it set?

Reply:
It's set in the test.yml (I renamed this yml file). I changed the #comments to make it clear.